apps/qubit/modules/search/actions/autocompleteAction.class.php

```diff
@@ -76,7 +76,10 @@ public function execute($request)
 
         foreach ($items as $item) {
             $search = new \Elastica\Search($client);
-            $search->addIndex($index)->addType($index->getType($item['type']));
+            foreach ($indices as $type => $index) {
+                $itemType = QubitSearch::getInstance()->index->getIndexTypeName($item['type']);
+                $search->addIndex($index)->addType($index->getType($itemType));
```

Comment:
Instead of using a different type on each index, I'd suggest to use the dummy type name `_doc`, as it will be used in 7.x:

https://www.elastic.co/blog/removal-of-mapping-types-elasticsearch#schedule

Using the same type for all indices should simplify many places modified in this PR.